### PR TITLE
added DB Adapters tests

### PR DESCRIPTION
**Pull request by laurencereeds**
Added the DB Adapters testing code + made a few corrections on the DB Adapters.

### CODE REVIEW

**Review by tillyninjaspace — COMMENTED** [review]
Hi Laurence, this looks good. I'm going to have Zach take a second look to see too and have him approve in case he catches something that I didn't.

File: db/index.js [hunk 1]
```diff
@@ -54,11 +56,71 @@ async function createProduct(product) {
...
+async function createUser({firstName, lastName, email, imageurl, username, password, isAdmin}) {
+  const hashedPassword = await bcrypt.hash(password, SALT_COUNT);
+  try {
+      const { rows: [user] } = await client.query(`
+      INSERT INTO users("firstName", "lastName", email, imageurl, username, password, "isAdmin")
+      VALUES($1, $2, $3, $4, $5, $6, $7)
+      RETURNING id, username;
+  `, [firstName, lastName, email, imageurl, username, hashedPassword, isAdmin]);
+  console.log('createdUser:', user)
+  // delete user.password
```
Comment (zachgarrett24) [inline db/index.js]: Let's double check whether or not we should return password or not. If not let's remove commented out code.

File: db/init_db.js [hunk 2]
```diff
@@ -71,32 +77,104 @@ async function buildTables() {
...
   try {
     console.log('Starting to create products...')
     const productsToCreate = [
       { name: 'camo masks', description: 'adult sizes', price: 5, imageURL: 'https://gracious-mcnulty-e733ac.netlify.app/images/sewingmachine.jpg', inStock: true, category: 'adults'},
       { name: 'Golden Gopher masks', description: 'for kids', price: 2, imageURL: 'https://gracious-mcnulty-e733ac.netlify.app/images/kidsmasks.jpg', inStock: true, category: 'kids'}
       ]
+
+      // const usersToCreate = [
+      //   {firstName: 'Berto', lastName: 'One', email: 'albert@gmail.com', imageurl: 'https://picsum.photos/200', username: 'albert', password: 'berto99', isAdmin: false},
+      //   {firstName: 'Sandy', lastName: 'Two', email: 'sandra@gmail.com', imageurl: 'https://picsum.photos/200', username: 'sandy', password: 'glamgal', isAdmin: false}
```
Comment (zachgarrett24) [inline db/init_db.js]: Remove commented out code.

**Review by zachgarrett24 — COMMENTED** [review]
It looks good to me. I suggest to just remove the commented out code to keep our dev branch clean.

**Review by tillyninjaspace — APPROVED** [review]
I see Laurence made the console.log removed change. Zach, you can go ahead and approve and merge if this all looks to you.

**Review by zachgarrett24 — APPROVED** [review]

**Pull request merged**